src/StackAdmin/Azs.Backup.Admin/Tests/src/BackupLocation.Tests.ps1

```diff
@@ -175,7 +185,19 @@ InModuleScope Azs.Backup.Admin {
 
             $backup = Start-AzsBackup -ResourceGroupName $global:ResourceGroupName -Location $global:Location -Force
             $backup 					| Should Not Be $Null
-            Restore-AzsBackup -ResourceGroupName $global:ResourceGroupName -Location $global:Location -Name $backup.Name -Force
+
+            try
+            {
+                [System.IO.File]::WriteAllBytes($global:decryptionCertPath, [System.Convert]::FromBase64String($global:decryptionCertBase64))
+                Restore-AzsBackup -ResourceGroupName $global:ResourceGroupName -Location $global:Location -Name $backup.Name -DecryptionCertPath $global:decryptionCertPath -DecryptionCertPassword $global:decryptionCertPassword -Force
```

Comment:
The Restore-AzsBackup should be output the try catch.  If it fails you want to actually catch it!